package.json

```diff
@@ -98,6 +98,7 @@
     "eslint-plugin-react-hooks": "4.6.0",
     "jest": "^29.4.1",
     "jest-environment-jsdom": "^29.4.1",
+    "next-sitemap": "^4.2.3",
```

Comment:
@kylegach Why did you need to use this package? I though Next.js managed sitemap on their own now. https://nextjs.org/docs/app/api-reference/file-conventions/metadata/sitemap#generating-a-sitemap-using-code-js-ts

Reply:
I tried that approach and it didn't work in production. 😭 `next dev` & `next start` worked fine, local and Netlify builds did not. I tried for a couple hours. It seems to be a [not-uncommon problem](https://www.google.com/search?q=nextjs+%22sitemap.ts%22+production+-%22next-sitemap%22). You can actually see that your prior attempt using `sitemap.ts` never worked either: https://deploy-preview-21--storybook-web.netlify.app/sitemap.xml (linking to the previous deploy preview for posterity).